### DISCUSSION TITLE
Allow MultiGraph and MultiDiGraph conversion

### DISCUSSION THREAD

**Discussion by vakker** [pyg-team/pytorch_geometric#3239] [Ideas]
Hi,

I have a a multi graph problem defined as an NX graph, but using `from_networkx` throws an error.

Changing [this](https://github.com/pyg-team/pytorch_geometric/blob/74245f3a680c1f6fd1944623e47d9e677b43e827/torch_geometric/utils/convert.py#L167) line to the following resolves this issue:
```
data['edge_index'] = edge_index[:2].view(2, -1)
```

However, `to_networkx` might be less trivial as it would need to check if all the edges are unique, or maybe `Data` could include a property `is_multi` (similarly to `is_directed`).

I patched this locally so it works with my stuff, but I'm happy to open a PR and work out the `to_networkx` part if needed.

**Comment by rusty1s** [comment]
This is interesting. `from_networkx` and `to_network`x should definitely support multi graphs. Feel free to open a PR for it :)
